code/src/App.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
+import { MovieList } from 'Components/List'
+import { Details } from 'Components/Details'
+import './index.css';
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <Router>
+      <Routes>
+        <Route path="/" element={<MovieList />} />
+        <Route path="/movies/:id" element={<Details />} />
```

Comment:
In the `Routes` component, you should add a fallback `Route` that renders a "Page Not Found" component in case the requested URL does not match any of the defined `Routes`.